lib/public/views/QcFlags/details/QcFlagDetailsPage.js

```diff
@@ -0,0 +1,107 @@
+import { deleteButton } from '../../../components/common/form/deleteButton.js';
+import { detailsList } from '../../../components/Detail/detailsList.js';
+import spinner from '../../../components/common/spinner.js';
+
+/**
+ * Create panel with QC flag details display
+ *
+ * @param {QcFlagDetailsModel} detailsModel model
+ * @return {Component} panel
+ */
+const qcFlagDetailsComponent = (detailsModel) => {
+    const { qcFlag: remoteQcFlag } = detailsModel;
+
+    const commentPanel =
+    h(
+        PanelComponent,
+        [
+            remoteQcFlag.match({
+                Success: (qcFlag) => qcFlag.comment
+                    ? [
+                        h(LabelPanelHeaderComponent, 'Comment'),
+                        markdownDisplay(
+                            qcFlag.comment,
+                            {
+                                classes: 'w-100',
+                            },
+                        ),
+                    ]
+                    : h('em', 'No comment'),
+                Other: () => null,
+            }),
+        ],
+    );
+
+    const leftPanel = [
+        remoteQcFlag.match({
+            Success: (qcFlag) => h('.flex-column', [
+                detailsList(
+                    qcFlagDetailsConfiguration(detailsModel),
+                    qcFlag,
+                    { selector: 'qc-flag-details' },
+                ),
+            ]),
+            Other: () => null,
+        }),
+        commentPanel,
+    ];
+
+    const rightPanel = [
+        detailsModel.deleteResult.match({
+            Failure: (errors) => errorAlert(errors),
+            Other: () => null,
+        }),
+        h('.pv3', detailsModel.deleteResult.match({
+            Loading: () => deleteButton(null, 'Sending...'),
+            Success: () => deleteButton(null, 'Sent!'),
+            Failure: () => deleteButton(() => detailsModel.delete()),
+            NotAsked: () => deleteButton(() => detailsModel.delete()),
+        })),
+    ];
+
+    return [
+        h('.w-50.flex-column.g3', leftPanel),
+        h('.w-50.flex-column.g3.items-center', rightPanel),
+    ];
+};
+
+/**
+ * Render QC flag details page
+ *
+ * @param {Model} model The overall model object.
+ * @returns {Component} The details page
+ */
+export const QcFlagDetailPage = ({ qcFlags: { detailsModel, forDataPassOverviewModel, forSimulationPassOverviewModel } }) => h('', {
+    onremove: () => {
+        forDataPassOverviewModel.reset();
+        forSimulationPassOverviewModel.reset();
+    },
+}, [
+    h('.flex-row.justify-between.items-center', h('.flex-row.g3', h('h2', 'QC Flag Details'))),
+    detailsModel.qcFlag.match({
+        Success: () => h('.w-100.flex-column', [h('.flex-row.g3.flex-grow', [qcFlagDetailsComponent(detailsModel)])]),
+        Loading: () => spinner(),
+        NotAsked: () => errorAlert([{ title: 'No data', detail: 'No QC flag data was asked for' }]),
+        Failure: (errors) => errorAlert(errors),
+    }),
+]);
```

Comment:
Try to rework indentation, for example try to put params on new lines when you have wrapped object props or array items

Reply:
refactored, removed unnecessary classes